egov-persister/evidence-persister.yml

```
@@ -132,4 +134,4 @@ serviceMaps:
               dbType: JSONB
             - jsonPath: $.artifact.auditdetails.lastModifiedBy
             - jsonPath: $.artifact.auditdetails.lastModifiedTime
-            - jsonPath: $.artifact.artifactNumber
+            - jsonPath: $.artifact.artifactNumber
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)